### DISCUSSION TITLE
paper/%ED%94%84%EB%A0%88%EC%A7%81-%EC%88%98%EC%97%85-%EC%A7%84%ED%96%89

### DISCUSSION THREAD

**Discussion by giscus** [ysw421/blog#7] [General]
# paper/%ED%94%84%EB%A0%88%EC%A7%81-%EC%88%98%EC%97%85-%EC%A7%84%ED%96%89

안녕하세요, 환영합니다!

https://www.siwonsw.com/paper/%ED%94%84%EB%A0%88%EC%A7%81-%EC%88%98%EC%97%85-%EC%A7%84%ED%96%89

**Comment by siwonyun** [comment]
# 댓글 테스트입니다 :)
###### 테스트!
💜💜💜💜💜

**Reply by siwonyun** [reply]
https://github.com/ysw421/My-blog/discussions/7 에서 자신의 글을 수정할 수 있어요!
